pkg/crypto/rings/client.go

```diff
@@ -301,6 +294,20 @@ func GetRingAddressesAtBlock(
 ) []string {
 	// Get the target session end height at which we want to get the active delegations.
 	targetSessionEndHeight := uint64(shared.GetSessionEndHeight(sharedParams, blockHeight))
+
+	return GetRingAddressesAtSessionEndHeight(app, targetSessionEndHeight)
+}
+
+// GetRingAddressesAtSessionEndHeight returns the active gateway addresses that need to be
+// used to construct the ring in order to validate that the given app should pay for.
```

Comment:
The sentence seems to be incomplete

```suggestion
// used to construct the ring in order to validate that the given app should pay for relays
// serviced in the session given by targetSessionEndHeight.
```